### PR TITLE
Add new bot events

### PR DESCRIPTION
**Pull request by clearab**
Adds recently added bot events.

### CODE REVIEW

File: msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md [hunk 1]
```diff
@@ -29,9 +34,12 @@ The following table shows a list of Teams conversation update events, with links
 | channel created     | channelCreated    | OnTeamsChannelCreatedAsync | [A channel was created](#channel-created) | Team |
 | channel renamed     | channelRenamed    | OnTeamsChannelRenamedAsync | [A channel was renamed](#channel-renamed) | Team |
 | channel deleted     | channelDeleted    | OnTeamsChannelDeletedAsync | [A channel was deleted](#channel-deleted) | Team |
-| team members added   | teamMemberAdded   | OnTeamsMembersAddedAsync   | [A Member added to team](#team-members-added)   | All |
-| team members removed | teamMemberRemoved | OnTeamsMembersRemovedAsync | [A Member was removed from team](#team-members-removed) | groupChat & team |
+| channel restored    | channelRestored    | OnTeamsChannelRestoredAsync | [A channel was restored](#channel-deleted) | Team |
+| members added   | membersAdded   | OnTeamsMembersAddedAsync   | [A member added](#team-members-added)   | All |
+| members removed | membersRemoved | OnTeamsMembersRemovedAsync | [A member was removed](#team-members-removed) | groupChat & team |
```
Comment (KirtiPereira) [inline msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md:39]:
```Suggestion
| members removed | membersRemoved | OnTeamsMembersRemovedAsync | [A member was removed](#team-members-removed) | groupChat and Team |
```

File: msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md [hunk 2]
```diff
@@ -282,6 +290,93 @@ async def on_teams_channel_deleted(
 
 * * *
 
+### Channel restored
+
+The channel restored event is sent to your bot whenever a channel that was previously deleted is restored in a team your bot is installed in.
```
Comment (KirtiPereira) [inline msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md:295]:
```Suggestion
The channel restored event is sent to your bot whenever a channel that was previously deleted is restored in a team that your bot is already installed in.
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
Reviewed the file. Please incorporate and verify the comments.
Thank you.

File: msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md (hunk 1 above)
Comment (clearab) [reply msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md:39]: This is incorrect, a "team" is the organizational construct inside "Teams".

Comment (KirtiPereira) [reply msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md:39]: Thank you for the clarification. We will go with the original content 'team'.

**Review by KirtiPereira — APPROVED** [review]
Thanks for incorporating the changes. Approved for merging.

**Review by surbhigupta12 — APPROVED** [review]
Thank you for incorporating the comments. Approving and merging this PR.

**Pull request merged**